frontend/src/components/worksheets/BundleDetail/BundleFieldTable/BundleFieldRow.jsx

```diff
@@ -27,43 +27,22 @@ class BundleFieldRow extends React.Component {
         super(props);
     }
 
-    /**
-     * Returns true if this field should be hidden.
-     * We should only show fields that either have values or are editable.
-     *
-     * @returns {bool}
-     */
-    checkHideRow() {
-        const field = this.props.field || {};
-        const value = this.props.value || field?.value;
-        if (!field.editable) {
-            if (!value) {
-                return true;
-            }
-            if (field.type === 'list' && (!value.length || !value[0])) {
-                return true;
-            }
+    getValue(field) {
+        const value = this.props.value || field.value;
+        if (value || value === false) {
```

Comment:
This looks strange - should this be `[false, true].includes(value)`?

Reply:
mm not quite. We want to return anything truthy and also `false`.

Reply:
actually it looks like bools get changed to a string, so I can remove the `false` check 👍